### PR TITLE
tests for most of the util functions

### PR DESCRIPTION
**Pull request by urbantumbleweed**
#### What's this PR do?

It adds tests for the Util.
#### What are the important parts of the code?

take a look at test/utils.js
#### How should this be tested by the reviewer?

download branch, npm install && npm run test

### CODE REVIEW

**Comment by urbantumbleweed** [comment]
I updated with tests for 4 additional files.

File: src/reducers/__tests__/content.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      const initial = '';
+      const action = { type: UPDATE_CONTENT, payload: payload };
+      const state = content(initial, action);
+      expect(state).toEqual(payload.content);
+      done();
+    });
+
+    it('even if the initial state is undefined', function(done){
+      const initial = undefined;
+      const action = { type: 'UPDATE_CONTENT', payload: payload };
```
Comment (phated) [inline src/reducers/__tests__/content.js]: use constant that was imported

File: src/reducers/__tests__/cwd.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const expect = require('expect');
+
+const cwd = require('../cwd');
+const { CHANGE_DIRECTORY } = require('../../constants');
+
+const payload = {
```
Comment (phated) [inline src/reducers/__tests__/cwd.js]: define inside the first describe?

File: src/reducers/__tests__/directory.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+const expect = require('expect');
+
+const directory = require('../directory');
+const {
+  SAVE_FILE,
+  DELETE_FILE,
+  CHANGE_DIRECTORY,
+  REFRESH_DIRECTORY
+  } = require('../../constants');
```
Comment (phated) [inline src/reducers/__tests__/directory.js]: this should be outdented

File: src/reducers/__tests__/directory.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+describe('#directory', function(){
+    const payload = {
```
Comment (phated) [inline src/reducers/__tests__/directory.js]: weird indentation, should only be 2

File: src/reducers/__tests__/directory.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+      listing: [
+        'file1',
+        'file2'
+      ]
+    };
+
+  it('returns empty array when initial state is undefined and type does not match', function(done){
+    const initial = undefined;
+    const type = undefined;
```
Comment (phated) [inline src/reducers/__tests__/directory.js]: i don't think type should be made a variable in this file

**Comment by urbantumbleweed** [comment]
I have completed tests for the methods directory.  There is one block in 'list-dir.js' that I am not sure how to test.  It is a case that returns an object that is not a File or a Directory in the target directory.

**Pull request closed without merging**